wsgi/odyssy/events/models.py

```diff
@@ -0,0 +1,28 @@
+from django.db import models
+
+
+class Event(models.Model):
+    start_time = models.DateField()
+    end_time = models.DateField()
+    event_date = models.DateField()
```

Comment:
The thing was there for announcements. In this case, this would imply the start and end time of the event, and not the period for which it stays on the website.